src/components/observer/resize_observer/resize_observer.stories.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useState } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { EuiPaddingSize } from '../../../global_styling';
+import { EuiText } from '../../text';
+import { EuiCode } from '../../code';
+import { EuiSpacer } from '../../spacer';
+import { EuiButton, EuiButtonEmpty } from '../../button';
+import { EuiPanel } from '../../panel';
+import { EuiResizeObserver, EuiResizeObserverProps } from './resize_observer';
+
+const meta: Meta<EuiResizeObserverProps> = {
+  title: 'Utilities/EuiResizeObserver',
+  component: EuiResizeObserver,
+};
+
+export default meta;
+type Story = StoryObj<EuiResizeObserverProps>;
+
+const StatefulPlayground = (props: EuiResizeObserverProps) => {
+  const { onResize, ...rest } = props;
```

Comment:
```suggestion
const StatefulPlayground = ({ onResize, ...props }: EuiResizeObserverProps) => {
```

Super optional syntax/destructuring suggestion, since we're not using `props` again anywhere else